core/migrations/14_deploy_register_creator.js

```diff
@@ -10,7 +10,7 @@ module.exports = async function(deployer, network, accounts) {
   const tokenizedDerivativeCreator = await TokenizedDerivativeCreator.deployed();
```

Comment:
what about tokenized derivative? is this getting deprecated

Reply:
Two things:
1. I wanted to only cover DVM code, since that will be frozen soon.
2. TD will be deprecated in the next few months, so it's not worth doing a lot of work to change the name IMO.